src/liballoc/char.rs

```diff
@@ -0,0 +1,26 @@
+/// Implements the `+` operator for concatenating two `char`s together.
+///
+/// This operation results into a new string being allocated with copies of the two `char`s in
+/// the respective order.
+///
+/// # Examples
+///
+/// ```
+/// let a = char::from("🎈");
+/// let b = char::from('🎉');
```

Comment:
```
let a = '🎈';
let b = '🎉';
```

Reply:
Thanks @hellow554. Please check now.

Reply:
it is better now, but you didn't fix what @hellow554 suggested. :wink: 
You should remove `char::from(`

Reply:
@BO41 Should that be done just in the comments part or everywhere else as well?

Reply:
consistency is always good. So I assume everywhere

Reply:
Okay thanks @BO41 ! I did this for the tests as well. 😀